### ISSUE TITLE
bug: Muted Text is difficult to read

### ISSUE THREAD

**Issue by edent** [hugo-themes/toha#1083]
## Expected Behavior
All text should be readable - and ideally meet WCAG accessibility guidelines.

## Current Behavior
The ` muted-text-color` is `#64748b` which is too low contrast.

<img width="347" height="246" alt="Screenshot showing poor contrast." src="https://github.com/user-attachments/assets/2111ed41-307f-4344-858a-c7f03f598e87" />

## Possible Solution

Brightening the text to `#95aed1` will keep the colour very similar and will increase the contrast to WCAG AAA level.

## Steps to Reproduce
1. Run an accessibility check using browser's built-in tools.
2. Or, find a friend with visual impairments.

## Screenshot

<img width="347" height="246" alt="Screenshot showing AAA contrast." src="https://github.com/user-attachments/assets/a85dfbc2-af43-44f8-8ce0-6b52ad7e4ce8" />

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari 11
- [x] Safari 10
- [x] IE 11

## Detailed Description

I'll send a PR to update that colour.
